packages/ui-components/Common/CircularIcon/index.tsx

```diff
@@ -0,0 +1,33 @@
+import classNames from 'classnames';
+
+import styles from './index.module.css';
+
+export type CircularIconProps = {
+  kind:
+    | 'event'
+    | 'method'
+    | 'property'
+    | 'class'
+    | 'module'
+    | 'classMethod'
+    | 'ctor';
+  size?: 'lg' | 'md' | 'sm';
+};
+
+const symbolMap = {
+  event: 'E',
+  method: 'M',
+  property: 'P',
+  class: 'C',
+  module: 'M',
+  classMethod: 'S',
+  ctor: 'C',
+} as const;
+
+export default function CircularIcon({ kind, size = 'md' }: CircularIconProps) {
+  return (
+    <div className={classNames(styles.icon, styles[size], styles[kind])}>
```

Comment:
```suggestion
    <div className={classNames(styles.icon, styles[size], styles[kind])} title={kind}>
```

it's will allow the user to know what doesn't it mean by hovering